libs/kg-commons/src/main/java/eu/ebrains/kg/commons/jsonld/JsonLdDoc.java

```diff
@@ -222,4 +221,13 @@ private void replaceNamespace(String oldNamespace, String newNamespace, Map curr
             }
         }
     }
+
+    public void removeAllPropertiesWhenNoPayload() {
```

Comment:
Same as for DynamicJson - let's try to keep them out of the generic classes and add them to the NormalizedJsonLd class